modules/store/spec/store.spec.ts

```diff
@@ -534,24 +534,44 @@ describe('ngRx Store', () => {
       expect(metaReducerSpy2).toHaveBeenCalled();
     });
 
-    it('should create a meta reducer for feature and call it with the expected reducer', () => {
+    it('should create a meta reducer for feature and call it with the expected reducer', (done: DoneFn) => {
+      const counterReducer2Spy = jasmine
+        .createSpy('counterReducer2Spy', counterReducer2)
+        .and.callThrough();
+
       TestBed.configureTestingModule({
         imports: [
           StoreModule.forRoot({}),
           StoreModule.forFeature('counter1', counterReducer, {
             metaReducers: [metaReducerContainer.metaReducer1],
           }),
-          StoreModule.forFeature('counter2', counterReducer2, {
-            metaReducers: [metaReducerContainer.metaReducer2],
-          }),
+          StoreModule.forFeature(
+            'counter2',
+            { counter: counterReducer2Spy },
+            {
+              initialState: { counter: 2 },
+              metaReducers: [metaReducerContainer.metaReducer2],
+            }
+          ),
         ],
       });
+
       const mockStore = TestBed.get(Store);
       const action = { type: INCREMENT };
+
       mockStore.dispatch(action);
 
       expect(metaReducerSpy1).toHaveBeenCalledWith(counterReducer);
-      expect(metaReducerSpy2).toHaveBeenCalledWith(counterReducer2);
+      expect(metaReducerSpy2).toHaveBeenCalled();
+      expect(counterReducer2Spy).toHaveBeenCalledWith(2, action);
```

Comment:
We should let this test as it was, to check if the right meta reducer was called with the reducer.
We should create a new test to verify the initial state.

Reply:
Ok, should the test should be as part of the metareducer suite?